ocpp/charge_point.py

```diff
@@ -218,7 +218,7 @@ async def _handle_call(self, msg):
             # after handler
             response = handler(**snake_case_payload)
             if inspect.isawaitable(response):
-                await response
+                asyncio.ensure_future(response)
```

Comment:
cant we use the higher level API (asyncio.create_task)?

Reply:
Good suggestion. Unfortunately  `asyncio.create_task()` was added in Python 3.7, while we also support Python 3.6.

Reply:
True. Then this is resolved